src/DimensionValidator.php

```diff
@@ -15,57 +15,43 @@
 class DimensionValidator extends ValueValidatorObject {
 
 	/**
-	 * @since 0.1
-	 *
-	 * @var boolean
+	 * @var bool
 	 */
-	protected $allowAuto = false;
+	private $allowAuto = false;
```

Comment:
this is a breaking change, cannot be done before next breaking rel, and might not make sense in there

Wikibase is not the only user of this lib btw